src/Accessibility/Announcer/utils.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2020 Metrological
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Code from: https://github.com/jashkenas/underscore is
+ * Copyright (c) 2009-2022 Jeremy Ashkenas, Julian Gonggrijp, and DocumentCloud and Investigative Reporters & Editors
+ * Licensed under the MIT License based off:
+ * http://unscriptable.com/2009/03/20/debouncing-javascript-methods/ which is:
+ * Copyright (c) 2007-2009 unscriptable.com and John M. Hann
+ * Licensed under the MIT License (with X11 advertising exception)
+ */
+
+export function getElmName(elm) {
+  return elm.ref || elm.constructor.name
+}
+
+/**
+ * Returns a function, that, as long as it continues to be invoked, will not
+ * be triggered. The function will be called after it stops being called for
+ * N milliseconds. If `immediate` is passed, trigger the function on the
+ * leading edge, instead of the trailing. The function also has a property 'clear'
+ * that is a function which will clear the timer to prevent previously scheduled executions.
+ *
+ * @source underscore.js
+ * @see http://unscriptable.com/2009/03/20/debouncing-javascript-methods/
+ * @param {Function} function to wrap
+ * @param {Number} timeout in ms (`100`)
+ * @param {Boolean} whether to execute at the beginning (`false`)
+ * @api public
+ */
+export function debounce(func, wait, immediate) {
```

Comment:
isn't this one available as an installable npm package, instead of copying over the code? Alternatively we could use the lodash equivalent, which I'm pretty sure has separate packages for each individual function